<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverWidget</class>
 <widget class="QWidget" name="SolverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="cinolib::GLcanvas" name="canvas" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Solver Options</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Equation</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="solver_cb">
              <item>
               <property name="text">
                <string>0 - Constant ground-truth solution</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 - Linear ground-truth solution</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2 - Quadratic ground-truth solution</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3 - Cubic ground-truth solution</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4 - C-inf, homogeneous Dirichlet BC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5 - C-inf, non homogeneous Dirichlet BC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6 - C-inf, non homogeneous Franke function</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Degree</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Other parameters...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="run_btn">
           <property name="text">
            <string> Run Solver</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/icons/img/run_icon.png</normaloff>:/icons/img/run_icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="t_slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextEdit" name="log_text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cinolib::GLcanvas</class>
   <extends>QWidget</extends>
   <header>cinolib/gui/qt/glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
